### PR TITLE
Add disconnect method

### LINKED ISSUE #3: error: 'class WiFiClass' has no member named 'disconnect'
Hi,

when using WiFiST, we cannot call the disconnect API listed here:
https://www.arduino.cc/en/Reference/WiFiDisconnect

Is it possible to add this service?

I'd like to use the WiFi like this:
SPIClass SPI_3(PC12, PC11, PC10);
WiFiClass WiFi(&SPI_3, PE0, PE1, PE8, PB13);
...
status = WiFi.begin(ssid, password);
...
WiFi.disconnect();

Thanks & Regards,

### PR DESCRIPTION
**Pull request by ghost**
Fix #3 
Disconnect function has been added.

### CODE REVIEW

**Review by LMESTM — APPROVED** [review]
Looks good - has this been tested successfully ?

**Comment by ghost** [comment]
Yes, it has been.
Tested multiple connect/disconnect loop with success.

File: src/WiFiST.cpp [hunk 1]
```diff
@@ -215,6 +215,17 @@ int WiFiClass::begin(char* ssid, const char *passphrase)
   return WL_CONNECT_FAILED;
 }
 
+/*
+* @brief  Disconnect the WiFi shield from the current network
+* @param  none
+* @retval none
+*/
+void WiFiClass::disconnect(void)
+{
```
Comment (fpistm) [inline src/WiFiST.cpp:224]: What happen if not connected before?

Comment (ghost) [reply src/WiFiST.cpp:224]: You can call `disconnect()` before `begin()` without trouble.
But I added a control on the wifi status to be sure the init of the device is done at least once.

**Review by fpistm — APPROVED** [review]
Wait @fred-r feedback

**Comment by fpistm** [comment]
Hi @fred-r
Have you tested this?

**Review by VVESTM — COMMENTED** [review]
Is it possible to update some examples also ?

Else, OK for me.
I have tested to add disconnect function to "ConnectWithWPA" like example.

**Review by fpistm — APPROVED** [review]

**Comment by fpistm** [comment]
An example could be added later

**Pull request merged**